guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -102,9 +102,9 @@ Blocks employ templates to generate {% glossarytooltip a2aff425-07dd-4bd6-9671-2
 
 To pass parameters use the <a href="#argument">`<argument></argument>`</a> instruction. 
 
-### &lt;container&gt; {#fedg_layout_xml-instruc_ex_cont}
```

Comment:
I would not eliminate the "<" and the ">" in this title, it references a node inside layout.